### ISSUE TITLE
PPSSP Core behavour with Vulcan drivers on Android

### ISSUE THREAD

**Issue by farfenkugell** [libretro/RetroArch#13489]
Hello,

Unsure if this is a CORE issue, driver issue, or RetroArch issue or a combination of both.
If you use PPSSPP core on Android device, for me it was Android 11 devices and set RetroArch to Vulcan drivers the rotation of wrong.

https://i.imgur.com/JOkcY6y.jpg

https://i.imgur.com/sT1nJUT.jpg

Tried on two devices, the behavior was the same.

**Comment by SirPrimalform** [comment]
Yup, this is still broken. Is the PPSSPP core abandoned? I love RA, but the PPSSPP core is unbelievably far behind the original stand-alone version.

**Comment by hizzlekizzle** [comment]
ppsspp-libretro builds from the upstream repo. it's always on the latest commit.

**Comment by SirPrimalform** [comment]
> ppsspp-libretro builds from the upstream repo. it's always on the latest commit.

But isn't it ~20000 commits behind standalone PPSSPP? If this is a bug from the master it may be one that was fixed a long time ago.

There are workarounds involving manual rotation settings in an override, but that completely messes up some shaders.

**Comment by hizzlekizzle** [comment]
No, it builds from the upstream repo. Our fork is no longer used, since the libretro-ization is in the upstream codebase.

**Comment by SirPrimalform** [comment]
Ah, thanks for the explanation.
